tools/lint-commits

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# Lint all commit messages that are newer than upstream/master if running
+# locally or the commits in the push or PR if in CircleCI.
+
+# The rules can be found in /.gitlint
+
+if [[ "
+$(git remote -v)
+" =~ '
+'([^[:space:]]*)[[:space:]]*(https://github\.com/|ssh://git@github\.com/|git@github\.com:)zulip/zulip(\.git|/)?\ \(fetch\)'
```

Comment:
Is this a straight copy from the zulip/zulip repo? I'd expect a note if so, at minimum in the commit text.

Reply:
Sure I will add an explicit commit description.

Reply:
This also needs to be:
```suggestion
'([^[:space:]]*)[[:space:]]*(https://github\.com/|ssh://git@github\.com/|git@github\.com:)zulip/python-zulip-api(\.git|/)?\ \(fetch\)'
```